docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
```

Comment:
nit: we should probably name this file `docker-compose.yaml` as per https://yaml.org/faq.html